app/conduct/conduct.js

```diff
@@ -0,0 +1,8 @@
+angular.module('gdgXBoomerang')
+.controller('ConductController', function ($http, $sce, Config, NavService) {
+    var vm = this;
+    vm.loading = true;
+    NavService.setNavTab(6);
+    vm.chapter = Config.name;
+    vm.lead = Config.lead;  
```

Comment:
We could get this from the organizer's API like in `organizers.js` and then put a list of the names on the page along with links to their G+ profiles.